lib/asciidoctor/pdf/ext/prawn/formatted_text/box.rb

```diff
@@ -56,7 +56,7 @@ def find_font_for_this_glyph char, current_font, current_font_opts = {}, fallbac
             (doc.instance_variable_get :@missing_chars) : (doc.instance_variable_set :@missing_chars, {})
         previous_fonts_checked = (missing_chars[char] ||= [])
         if previous_fonts_checked.empty? && !(previous_fonts_checked.include? fonts_checked)
-          logger.warn %(Could not locate the character `#{char}' in the following fonts: #{fonts_checked.join ', '})
+          logger.warn %(Could not locate the character `#{char}' (#{char.dump.delete_prefix('"').delete_suffix '"'}) in the following fonts: #{fonts_checked.join ', '})
```

Comment:
This is better done using unpack.

```suggestion
          logger.warn %(Could not locate the character `#{char}' (#{char.unpack('U*').map {|it| "\\u#{(it.to_s 16).rjust 4, '0'}" }.join}) in the following fonts: #{fonts_checked.join ', '})
```